js/packages/web/src/components/AppBar/index.tsx

```diff
@@ -120,7 +120,12 @@ export const AppBar = () => {
     <>
       <div id="mobile-navbar">
         <LogoLink />
-        <MetaplexMenu />
+        <div className="mobile-menu">
```

Comment:
What do you think of creating a separate component with its style file?

I.e. <MobileMenu>

Reply:
MobileNavbar

Reply:
Do you mean separate components for desktop and mobile menu?